packages/react-chat/src/components/Chat.tsx

```diff
@@ -83,6 +87,11 @@ export function Chat({ theme, community, fetchMetadata }: ChatProps) {
         description={community.description}
         publicKey="0xD95DBdaB08A9FED2D71ac9C3028AAc40905d8CF3"
       />
+      <PictureModal
+        isVisible={isModalVisible}
+        onClose={() => setIsModalVisible(false)}
```

Comment:
```suggestion
        isVisible={!!image}
        onClose={() => setImage('')}
```

we can use image as variable that decides when to show modal, so that we don't need unnecessary variables